plopfile.js

```diff
@@ -36,25 +34,19 @@ module.exports = function (plop) {
       },
     ],
     actions: function (data) {
-      const fileName = getFileName(data.name)
+      const appDirPath = data.appDir ? 'app-dir' : ''
+      let templatePath = `test/${
+        data.type === 'unit' ? 'unit' : 'e2e'
+      }/${appDirPath}/test-template`
+      let targetPath = `test/{{ type }}/${appDirPath}`
+
+      console.log(templatePath)
```

Comment:
Is this log expected? Looks like a debug log?

Reply:
yup